Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -5236,6 +5236,14 @@ on revIDEPropertySet pObjectList, pProperty, pValue, pLockUpdates
          
          # If stack name is changed, use new name when setting edited
          if tStack is tObject and pProperty is "name" then
+            if char 1 to 3 of pValue is "rev" then
+               answer warning "Using rev in the first three characters of a stack name is reserved for use by the LiveCode development environment and advanced users creating Plug-ins. If you use these characters, your stack may not behave as expected."
```

Comment:
@livecodepanos I think both of these answer dialogs need to have their messages wrapped in revIDELocalize. It's not really doing much now but I guess it will one day ;-)

Also perhaps have an opportunity to cancel here?

Eventually it would be nice to do away with the rev prefix thing and use `_ideoverride` everywhere but that's a long way off I think.